Project_source/app/src/main/java/android/clicker/school_live_simulator/Game.kt

```diff
@@ -1,4 +1,41 @@
 package android.clicker.school_live_simulator
 
-class Game {
+object Game {
+    private var difficulty_state: GameDifficultyState = NormalMode()
+    private var save_handlers: ArrayList<function> = ArrayListOf()
+    private var load_handlers: ArrayList<function> = ArrayListOf()
+
+    private lateinit var player: InGameWrapper
+//    game: Game
+    lateinit var game_date: GameDate
+    lateinit var game_status: GameStatus
+
+
+    fun init() {
+        TODO("Not yet implemented")
+    }
+    fun tick() {
+        TODO("Not yet implemented")
+    }
+
+    fun checkDefeat(school_performance: Int, happiness: Int, satiety: Int) {
+        TODO("Not yet implemented")
+    }
+    fun isActAvaliable(): Boolean {
+        TODO("Not yet implemented")
+    }
+
+    fun registerSaveHandler(handler: function) {
+        TODO("Not yet implemented")
+    }
+    fun registerLoadHandler(handler: function) {
+        TODO("Not yet implemented")
+    }
```

Comment:
Please, move this to a new issue about game saving methods (in case new issue about researching serialization features).
But now, I think, that it should be removed.

Reply:
I will comment that for now and add issue. Fixed.